src/main/resources/fine_tuning/models.yaml

```diff
@@ -174,30 +204,20 @@ models:
         lower: 0
         upper: 1
         pace: 0.1
-      b:
-        type: float
-        lower: 0
-        upper: 0.5
-        pace: 0.1
-      k1:
-        type: float
-        lower: 0.9
-        upper: 0.9
-        pace: 0.1
     expected:
       robust04:
         map:
-          best_avg: 0.2532
-          oracles_per_topic: 0.2921
-          2-fold: 0.2253
-          5-fold: 0.2524
+          best_avg: 0.3020
+          oracles_per_topic: 0.4343
```

Comment:
should be `0.4402`? Please check out the newest version of the paper to updated numbers

Reply:
I will leave it until we merge your PR? (actually you can update it?)

Reply:
I guess that's fine too